obspy/io/css/core.py

```diff
@@ -119,3 +151,59 @@ def _read_css(filename, **kwargs):
         tr = Trace(data, header=header)
         traces.append(tr)
     return Stream(traces=traces)
+    
+def _read_nnsa_kb_core(filename, **kwargs):
+    """
+    Reads a NNSA KB Core waveform file and returns a Stream object.
+    .. warning::
+        This function should NOT be called directly, it registers via the
+        ObsPy :func:`~obspy.core.stream.read` function, call this instead.
+    :type filename: str
+    :param filename: NNSA KB Core file to be read.
+    :rtype: :class:`~obspy.core.stream.Stream`
+    :returns: Stream with Traces specified by given file.
+    """
+    # read metafile with info on single traces
+    with open(filename, "rb") as fh:
+        lines = fh.readlines()
+    basedir = os.path.dirname(filename)
+    traces = []
+    # read single traces
+    for line in lines:
+        npts = int(line[80:88])
+        dirname = line[149:213].strip().decode()
+        filename = line[214:246].strip().decode()
+        filename = os.path.join(basedir, dirname, filename)
+        offset = int(line[247:257])
+        dtype = DTYPE[line[144:146]]
+        if isinstance(dtype, tuple):
+            read_fmt = np.dtype(dtype[0])
+            fmt = dtype[1]
+        else:
+            read_fmt = np.dtype(dtype)
+            fmt = read_fmt
+        with open(filename, "rb") as fh:
+            fh.seek(offset)
+            data = fh.read(read_fmt.itemsize * npts)
+            data = from_buffer(data, dtype=read_fmt)
+            data = np.require(data, dtype=fmt)
+        header = {}
+        header['station'] = line[0:6].strip().decode()
+        header['channel'] = line[7:15].strip().decode()
+        header['starttime'] = UTCDateTime(float(line[16:33]))
+        header['sampling_rate'] = float(line[89:100])
+        header['calib'] = float(line[101:117])
+        header['calper'] = float(line[118:134])
+        tr = Trace(data, header=header)
+        traces.append(tr)
+    return Stream(traces=traces)
+
+
+# Remove once 0.11 has been released.
+sys.modules[__name__] = DynamicAttributeImportRerouteModule(
+    name=__name__, doc=__doc__, locs=locals(),
+    original_module=sys.modules[__name__],
+    import_map={},
+    function_map={
+        "isCSS": "obspy.io.css.core._is_css",
+        "readCSS": "obspy.io.css.core._read_css"})
```

Comment:
This whole block comes from an old copy of this file and should be removed again, it's not in current master anymore.